#!/usr/bin/env python
PACKAGE="riptide_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()


#        Name            Type      Level  Description                       Default     Min       Max
gen.add( "Buoyant_Force",   double_t, 1,     "Force of Buoyancy",           419.2,      0,        1000)
gen.add( "Buoyancy_X_POS",  double_t, 1,     "Buoyancy X Pos.",             .0308,          -0.5,     0.5)
gen.add( "Buoyancy_Y_POS",  double_t, 1,     "Buoyancy Y Pos.",             .0005,         -0.5,     0.5)
gen.add( "Buoyancy_Z_POS",  double_t, 1,     "Buoyancy Z Pos.",             -.034,          -0.5,     0.5)
                 # PkgName  #NodeName         #Prefix for generated .h include file, e.g. ParametersConfig.py
exit(gen.generate(PACKAGE, "thruster_controller", "VehicleProperties"))
